CanaanAndAmy

```diff
@@ -0,0 +1,177 @@
+
+require "colorize"
+require "faker"
+
+class ASCII_display
+  attr_reader :ASCII_init, :displayed
+  def initialize
+    @ASCII_init = ["  @ . . @".green, "  ( --- )".green, " (  >__<  )".green, " ^^  ~~  ^^".green, " ~~~~~~~~~~~".green, "  ~~~~~~~~~".green]
+    @displayed = display_picture(@ASCII_init)
```

Comment:
probably not necessary to make `@displayed = display_picture()` and just call `.display_picture()` instead of `.displayed` on line 41